### DISCUSSION TITLE
Broken protein chain in out put

### DISCUSSION THREAD

**Discussion by DanielLevinLab** [RosettaCommons/rosetta#98] [RosettaScripts]
Hi,

I design a protein-ligand interaction with rosetta however the output is comping out as a broken chain while the whole sequence is intact, is there a known reason to what causes it?

The ligand is already placed in the binding pocket and there are no clashes

**Comment by ajasja** [comment]
Probably not enough information to answer this question. How did you redesign, what does the log say, are there any errors.

**Reply by DanielLevinLab** [reply]
I used a protocol called COMBS from degrado lab to design binding site and then I use rosetta to design the rest of the protein
I looked just now at the logs and here is a warning I get that I didn't see before hand:
core.optimization.AtomTreeMinimizer: [ WARNING ] ***************************************************************
core.optimization.AtomTreeMinimizer: [ WARNING ] ** WARNING: Non-ideal minimization used with a ScoreFunction **
core.optimization.AtomTreeMinimizer: [ WARNING ] **    which isn't set up for non-ideal minimization          **
core.optimization.AtomTreeMinimizer: [ WARNING ] ***************************************************************

**Comment by NooraAz** [comment]
It would be great if you could provide the **code** and **inputs**, or a link to them, so we can reproduce the error. My guess is your protein already has a jump. You can check this with `pose.annotated_sequence( )` and `pose.fold_tree()`.

**Reply by DanielLevinLab** [reply]
[Input.zip](https://github.com/user-attachments/files/16366352/Input.zip)

Thank you for the quick reply here is my input

**Reply by NooraAz** [reply]
Thanks! and your command line + flags?

**Reply by DanielLevinLab** [reply]
-parser:protocol flexbb_design_protocol_ligand_docking.xml
-overwrite
-nstruct 1
-ignore_zero_occupancy false
-s AB6H-5.pdb
-extra_res_fa AXA.params
-out:path:all output/
#-out:file:silent 1.out
-out:file:scorefile score.txt 
-packing:linmem_ig 10

/sci/labs/jshifman/sergeib/rosetta_bin_linux_2021.16.61629_bundle/main/source/bin/rosetta_scripts.static.linuxgccrelease -database /sci/labs/jshifman/sergeib/rosetta_bin_linux_2021.16.61629_bundle/main/database/

**Reply by NooraAz** [reply]
Sorry I couldn't reproduce your job, because some files like `no_met_thr_ser_asn.comp`, `netcharge.charge`, `resfile.txt`, `constraints.cst` ... are not provided. But looking at your script, I don't see any obvious faults and the input pose has no jumps as well.

Just a question to help you debugging. Does the break always happen at the same place? if yes, where?

**Reply by DanielLevinLab** [reply]
oh yeah these are additional files that are needed to roseta but since this is the first round I didn'y add constraints.

The breaks are random I think I guess you could just run relax protocol on in it will have the same effect and you will probably won't need anything else aside from the input

**Comment by roccomoretti** [comment]
The error with the "Non-ideal minimization used" is due to the `bondangle="true" bondlength="true"` that you're including in your FastRelax. The way to fix it is to either remove those entries, or alternatively use `ref2015_cart` instead of `ref2015` in your `ref15_1` definition. (Either will likely work for you.)

That change could indeed cause arbitrary "broken" backbone chains, as the bond lengths can vary and stretch without being properly penalized.
